src/test/python/find_projection.py

```diff
@@ -1,13 +1,5 @@
-from inspect import currentframe, getframeinfo
-from pathlib import Path
-import sys
-
-filename = getframeinfo(currentframe()).filename
```

Comment:
I know it's gone now, but `__file__` contains the same value as you were getting here.

Reply:
Now he tells me :)